src/EnergyPlus/HVACVariableRefrigerantFlow.cc

```diff
@@ -11867,7 +11864,13 @@ void VRFCondenserEquipment::CalcVRFCondenser_FluidTCtrl(EnergyPlusData &state)
                                                    CapMinTe + 8,
                                                    this->IUCondensingTemp - 5);
 
-        if ((Q_c_OU * C_cap_operation) <= CompEvaporatingCAPSpdMin) {
+        Real64 CompEvaporatingCAPSpdMaxCurrentTsuc = this->CoffEvapCap * this->RatedEvapCapacity *
+                                                     CurveValue(state, this->OUCoolingCAPFT(NumOfCompSpdInput), Tdischarge, this->EvaporatingTemp);
+        Real64 CompEvaporatingPWRSpdMaxCurrentTsuc = this->RatedCompPower * CurveValue(state, this->OUCoolingPWRFT(NumOfCompSpdInput), Tdischarge, this->EvaporatingTemp);
+        if ((Q_c_OU * C_cap_operation) > CompEvaporatingCAPSpdMaxCurrentTsuc + CompEvaporatingPWRSpdMaxCurrentTsuc) {
+            CompSpdActual = this->CompressorSpeed(NumOfCompSpdInput);
+            Ncomp = this->RatedCompPower * CurveValue(state, this->OUCoolingPWRFT(NumOfCompSpdInput), Tdischarge, this->EvaporatingTemp);
+        } else if ((Q_c_OU * C_cap_operation) <= CompEvaporatingCAPSpdMin) {
```

Comment:
I can't comment on the math here, but what if the curve result yields CompEvaporatingPWRSpdMaxCurrentTsuc that gives a value in the IF that is less than CompEvaporatingCAPSpdMin in the ELSE IF? Just pointing this out so you think about what that might mean.

Reply:
yeah, this part might also need some change as well. The intention was to match the calculation here (which computes the compressor speed) and previously in limitTUCapacity. Let me sort this through

Reply:
I added a error message to ensure that CompEvaporatingCAPSpdMin is less than CompEvaporatingCAPSpdMaxCurrentTsuc so the case when `Q_c_OU * C_cap_operation <= CompEvaporatingCAPSpdMin` and `Q_c_OU * C_cap_operation > CompEvaporatingCAPSpdMaxCurrentTsuc + CompEvaporatingPWRSpdMaxCurrentTsuc` won't occur.